pkg/api/types.go

```diff
@@ -1865,3 +1863,39 @@ type ComponentStatusList struct {
 
 	Items []ComponentStatus `json:"items"`
 }
+
+// SecurityContext holds security configuration that will be applied to a container.  SecurityContext
+// contains duplication of some existing fields from the Container resource.  These duplicate fields
+// will be populated based on the Container configuration if they are not set.  Defining them on
+// both the Container AND the SecurityContext will result in an error.
+type SecurityContext struct {
+	// Capabilities are the capabilities to add/drop when running the container
+	// Must match Container.Capabilities or be unset.  Will be defaulted to Container.Capabilities if left unset
```

Comment:
This was specifically for https://github.com/GoogleCloudPlatform/kubernetes/pull/7343#discussion-diff-29297074.  They can still be specified in json so it'll help if folks are looking at a specific version definition